test/evergreen.yml

```diff
@@ -130,6 +130,83 @@ functions:
               --enable-strict --enable-diagnostic --with-builtins=lz4,snappy,zlib
           fi
     - *make_wiredtiger
+  "compile wiredtiger docs":
+    - command: shell.exec
+      params:
+        working_dir: "wiredtiger/build_posix"
+        shell: bash
+        script: |
+          set -o errexit
+          set -o verbose
+
+          if [[ "${branch_name}" != "develop" ]]; then
+            echo "We only run documentation update task on the WiredTiger (develop) Evergreen project."
```

Comment:
(Here and below) minor wording change, add `the`: "We only run the documentation update task..."